test/test_insert_manager.rb

```diff
@@ -28,6 +28,25 @@ module Arel
         }
       end
 
+      it 'inserts multiple values' do
+        table = Table.new(:users)
+        manager = Arel::InsertManager.new
+        manager.into table
+
+        manager.columns << table[:id]
+        manager.columns << table[:name]
+
+        manager.values = manager.create_tuple([
+          manager.create_values(%w{ 1 david }),
+          manager.create_values(%w{ 2 kirs }),
+          manager.create_values(["3", Arel.sql('DEFAULT')], []),
```

Comment:
It's ignored because with multiple insert there's no way to accept the column names and order for every value.

With multiple order you'd have to set the columns first:

```ruby
manager.columns << table[:id]
manager.columns << table[:name]
```

And then pass the values.